dusty/constants.py

```diff
@@ -6,6 +7,10 @@
 
 EXAMPLE_SPECS_REPO = 'github.com/gamechanger/dusty-example-specs'
 
+DUSTY_CONFIG_BEGIN = '# BEGIN section for Dusty\n'
+DUSTY_CONFIG_END = '# END section for Dusty\n'
+DUSTY_CONFIG_REGEX = re.compile('\\{}.*\\{}'.format(DUSTY_CONFIG_BEGIN, DUSTY_CONFIG_END), flags=re.DOTALL | re.MULTILINE)
```

Comment:
The regex is a touch different than before since begin has a `\n` now, maybe make sure it's all still kosher?